arch/toolchains/github-ubuntu-gnu.cmake

```diff
@@ -26,3 +25,4 @@ set(ECBUILD_Fortran_FLAGS "${ECBUILD_Fortran_FLAGS} -fno-second-underscore")
 set(ECBUILD_Fortran_FLAGS "${ECBUILD_Fortran_FLAGS} -ffree-form")
```

Comment:
I realise this is not touched in this commit, but is this flags required? If so it is not really a platform-specific flag, but should be embedded in source, i.e. each gfortran compiler would suffer this.

Reply:
I'll try to find out, good point!